spec/requests/api/v1/conversations/index_spec.rb

```diff
@@ -0,0 +1,76 @@
+describe 'GET /api/v1/conversations', type: :request do
+  subject { get api_v1_conversations_path, headers: auth_headers, as: :json }
+
+  context 'when the request is valid' do
+    let!(:user) { create(:user) }
+    let!(:second_user) { create(:user) }
+    let!(:third_user) { create(:user) }
+    let(:conversation) { create(:conversation, user_from: user, user_to: second_user) }
+    let(:second_conversation) { create(:conversation, user_from: third_user, user_to: user) }
+    let!(:expected_conversations) { [conversation, second_conversation] }
+    let!(:expected_messages) { [conversation.messages, second_conversation.messages] }
+    let!(:user_message) { create(:message, user:, conversation:) }
+    let!(:response_message) { create(:message, user: second_user, conversation:) }
+    let!(:user_message2) { create(:message, user: third_user, conversation: second_conversation) }
+    let!(:response_message2) { create(:message, user:, conversation: second_conversation) }
+
+    it 'returns success' do
+      subject
+      expect(response).to have_http_status(:success)
+    end
+
+    context 'when the user is logged in' do
+      it 'returns the conversations list count' do
+        subject
+        expect(json[:conversations].count).to eq(2)
+      end
+    end
+
+    it 'returns the targets list' do
+      subject
+      expect(json[:conversations].as_json(except: %i[messages])).to eq(
+        expected_conversations.reverse.as_json(except: :updated_at)
```

Comment:
I think it would be easier to do 
`expect(json[:conversations].pluck(:id)).to eq expected_conversations.pluck(:id)` or similar